server/services/interventionsService.test.ts

```diff
@@ -1290,6 +1290,233 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
       expect(await interventionsService.getPccRegions(token)).toEqual(pccRegions)
     })
   })
+
+  describe('createDraftActionPlan', () => {
+    it('returns a newly created draft action plan', async () => {
+      const referralId = '81d754aa-d868-4347-9c0f-50690773014e'
+      await provider.addInteraction({
+        state: 'a caseworker has been assigned to a sent referral and an action plan can be created',
+        uponReceiving: 'a POST request to create a draft action plan',
+        withRequest: {
+          method: 'POST',
+          path: '/draft-action-plan',
+          headers: { Accept: 'application/json', Authorization: `Bearer ${token}` },
+          body: {
+            referralId,
+          },
+        },
+        willRespondWith: {
+          status: 201,
+          body: Matchers.like({
+            id: 'dfb64747-f658-40e0-a827-87b4b0bdcfed',
+            referralId: '81d754aa-d868-4347-9c0f-50690773014e',
+            numberOfSessions: null,
+            activities: null,
```

Comment:
empty list implies zero activities, null implies no activities specified. which do we want?

Reply:
Yeah this is the one thing I wasn't sure about and wanted to discuss at some point

Reply:
In what scenario do we need to distinguish between unspecified and zero elements?

Reply:
I think this is a fair point, this is less of an issue than the Referral form where we relied on `null` values to figure out the state of a `draftReferral`, but as this is a single journey (rather than e.g. a task list) I think we can probably just check for an empty array here. I do feel it's a bit nicer to work with than lots of `null` checks etc, but happy to chat further if anyone disagrees